README.md

```diff
@@ -143,7 +143,7 @@ If the test works, upload to the real deal:
 3.  The ``util`` directory contains the templates to format the release notes
     and the GitHub descriptions.
 4.  The script expects to find the current release number in either the Django
-    ``settings.py`` file or in the ``setup.py`` file in the project root
+    ``settings.py`` file or in the ``__init__.py`` file in the project root
```

Comment:
So we have two different scripts now `release.sh` and `release.py`. Do the still have the same behavior? It is very unclear in the documentation which one we talking about.

Reply:
They have the same behavior. `release.sh` will just call `release.py` and look up the repo url for the repo directory to pass it as an argument.